src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -550,4 +562,99 @@
      */
     RecipeRegistry getRecipeRegistry();
 
+    /**
+     * Gets the {@link BlockReason} with the given name.
+     * 
+     * @param name The name of the reason
+     * @return The {@link BlockReason} or Optional.absent() if not found
+     */
+    Optional<BlockReason> getBlockReason(String name);
```

Comment:
Isnt the actual block missing as param here?
Its required for getBlock inside BlockReason.